app/models/ext_management_system.rb

```diff
@@ -476,4 +477,11 @@ def stop_event_monitor_queue_on_credential_change
       self.stop_event_monitor_queue
     end
   end
+
+  def filtered_event_names
+    (
+     EventBlacklisting.per_model_filters(self).pluck(:event_name) +
+     event_blacklistings.where(:enabled => true).pluck(:event_name)
```

Comment:
If you are fixing @gmcculloug 's issues, then there is a minor indentation issue here as well.